app/api_alpha/views.py

```diff
@@ -1944,8 +1944,10 @@ def get_stats(request):
     api_calls_since_2024_count = APIRequestLog.objects.filter(
         requested_at__gte="2024-01-01T00:00:00Z"
     ).count()
-    contributions_count = Contribution.objects.count()
+    contributions_count = Contribution.objects.filter(report=True).count()
```

Comment:
On devrait faire la distinction signalement/édition je pense

```suggestion
    reports_count = Contribution.objects.filter(report=True).count()
```